stko/molecular/molecule_splitter/molecule_splitter.py

```diff
@@ -0,0 +1,157 @@
+"""
+Molecule Splitter
+=================
+
+#. :class:`.MoleculeSplitter`
+
+Class for splitting a molecule into many with new connectors.
+
+"""
+
+import logging
+from rdkit.Chem import AllChem as rdkit
+from itertools import combinations
+
+import stk
+
+logger = logging.getLogger(__name__)
+
+
+class MoleculeSplitter:
+    """
+    Split an stk.molecule into many with new functional groups.
```

Comment:
It sounds like this class is not just a splitter but also a molecule transformer. Could it be worth making two seperate classes; one class that does splittng and one for transforming functional groups?